evm-node.md

```diff
@@ -0,0 +1,149 @@
+```{.k .node}
+module EVM-NODE
+    imports EVM
+    imports K-REFLECTION
+    imports COLLECTIONS
+
+    syntax AccountCode ::= "#unloaded"
+
+    syntax Int ::= #getBalance(Int) [function, hook(MANTIS.getBalance)]
+                 | #getNonce(Int) [function, hook(MANTIS.getNonce)]
+    syntax Bool ::= #isCodeEmpty(Int) [function, hook(MANTIS.isCodeEmpty)]
+                  | #accountExists(Int) [function, hook(MANTIS.accountExists)]
```

Comment:
I'm not a huge fan of the hooks being `MANTIS.*`, because they shouldn't be Mantis specific, should they?